extensions/kind/src/extension.ts

```
@@ -190,6 +193,36 @@ async function searchKindClusters(provider: extensionApi.Provider) {
   updateClusters(provider, kindContainers);
 }
 
+type ImageInfo = { engineId: string; id: string; name: string };
+
+async function moveImage(image: ImageInfo) {
```

as the command will be registered globally, image parameter could be empty

For example, when command Palette is implemented, we'll be able to call that command. Thus, the command should prompt for an image from the current set of images in case image is not there

---

Not sure to get it do you ask for a change in this PR or is this something to be in a separate issue ?

---

I would say:
follow-up if you handle the optional case in a coming PR
and if no follow-up soon: probably need to throw an error if parameter is not provided